package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jvm-npm",
+  "version": "0.1.0",
+  "description": "Support for NPM module loading in Javascript runtimes on the JVM",
+  "main": "src/main/javascript/jvm-npm.js",
+  "directories": {
+    "example": "examples"
+  },
+  "scripts": {
+    "test": "mvn integration-test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nodyn/jvm-npm.git"
+  },
+  "author": "lance",
```

Comment:
Can you make this "Lance Ball <lball@redhat.com> (http://lanceball.com)" please?